test/acceptance/features/steps/steps.py

```diff
@@ -557,3 +557,30 @@ def update_binding_secret(context):
     output = openshift.apply(yaml.dump(secret_yaml), context.namespace.name)
     result = re.search(rf'.*{context.sb_secret}.*(created|unchanged|configured)', output)
     assert result is not None, f"Unable to apply YAML for binding secret '{context.sb_secret}': {output}"
+
+
+@given(u'CustomResourceDefinition backends.stable.example.com is available')
+def backends_crd(context):
```

Comment:
Could you please move the CRD YAML into the separate file under `test/acceptance/resources` and load it here from it, so it is not hard-coded?

Reply:
I can see that the actual CRD is already there as part of `test/acceptance/resources/backend-new-spec.operator.manifest.yaml`.